src/components/AppContainer.js

```diff
@@ -31,10 +31,18 @@ class AppContainer extends React.Component {
         this.setState(newState);
     }
 
-    handleFeedbackItem(value) {
+    async handleFeedbackItem(value) {
         let newState = this.state;
         newState.feedbacks = value;
-        this.setState(newState);        
+        this.setState(newState);
+        if (this.state.directory !== "") {
```

Comment:
If I understand correctly this will only change a `feedback.json` file if it exists but will not create one?

Reply:
Not quite. As long as a directory has been opened, the feedback object will be saved to that directory. If you haven't opened a directory then you won't know where to save.